packages/Radio/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+2020-10-27
+
+# Added
+
+- Moved moreGroupProps to be spread on Group component rather than children [@tristanjasper](https://github.com/tristanjasper).
```

Comment:
I'm not sure what convention we should follow here, but I think this should be clearly marked as a breaking change.